ansible/vtestbed.csv

```diff
@@ -1,5 +1,6 @@
 # conf-name,group-name,topo,ptf_image_name,ptf,ptf_ip,ptf_ipv6,server,vm_base,dut,inv_name,auto_recover,comment
 vms-kvm-t0,vms6-1,t0,docker-ptf,ptf-01,10.250.0.102/24,fec0::ffff:afa:2/64,server_1,VM0100,[vlab-01],veos_vtb,False,Tests virtual switch vm
+vms-kvm-sonic-t0,vms6-1,t0,docker-ptf,ptf-01,10.250.0.102/24,fec0::ffff:afa:2/64,server_1,VM0100,[vlab-01],vsonic_vtb,False,Tests virtual switch vm
```

Comment:
what if user wants to setup both t0 and sonic-t0? better to use a different Ip, use 10.250.0.112

Reply:
I removed this testbed because it equal with `vms-kvm-t0`